test/patient.test.js

```diff
@@ -0,0 +1,185 @@
+/* 
+Name: Dylan Bellinger
+Date: 11/11/2024 
+Description: Patient model unit tests.
+*/
+const Patient = require("../models/Patient");
+
+describe("Patient Test", () => {
+
+    let patientTest = null;
+    let date  = new Date();
+
+    patientTest = new Patient({
+        patient_id: "6ccd780c-baba-1026-9564-5b8c659018db",
+        section_id: null,
+        date_of_birth: date,
+        religion: "None",
+        full_name: "Sherlock Holmes",
+        weight: 150.0,
+        height: 5.6,
+        has_insurance: true,
+        has_advanced_directives: true,
+        allergies: {
+          allergy1: "Dust",
+          allergy2: "Peanuts",
+          allergy3: "Wheat",
+        },
+        emergency_contact_full_name: "John Watson",
+        emergency_contact_phone_number: "123-456-7890",
+        code_status: "FULL-CODE",
+        precautions: "CONTACT",
+      });
+
+      test("Necessary Patient fields are not null", () => {
```

Comment:
I think this should be `test("CreatesPatient_WhenAllConditionsMet_ReturnsObject", () => {`, don't you think?